ujt/transformers.py

```diff
@@ -0,0 +1,65 @@
+""" Module holding transformer functions.
+
+Transformer functions take a Dash-specific data structure or an intermediate data structure,
+make a change, and return the same type of data structure.
```

Comment:
what kind of changes can we expect to be made? 
is the invariant here the type of data structure being used or is there a difference between Dash and intermediate structures?

Reply:
Agreed, the docstring phrasing is confusing, I'll try to modify it.
 
For all the cases we currently have, transformer signatures are of the form: foo(cytoscape_elements, other_args) -> cytoscape_elements.
The changes currently being made include adding/removing nodes or edges, or modifying classes for styling.

I was trying to include the case where we have some generic type and a transformer bar(my_type, other_args) -> my_type in the docstring, but I don't have any code that falls into this category yet.